drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -708,14 +721,29 @@ static inline void unicam_schedule_next_buffer(struct unicam_node *node)
 	unicam_wr_dma_addr(dev, addr, node->pad_id);
 }
 
+static inline void unicam_schedule_dummy_buffer(struct unicam_node *node)
+{
+	struct unicam_device *dev = node->dev;
+	dma_addr_t addr = node->dummy_buf_dma_addr;
+
+	if (node->pad_id == IMAGE_PAD) {
+		reg_write(&dev->cfg, UNICAM_IBSA0, addr);
```

Comment:
This is so close to `unicam_wr_dma_addr`, can't you make use of it?

Reply:
Done.